src/main/java/uk/gov/companieshouse/api/accounts/controller/DebtorsController.java

```diff
@@ -78,6 +79,14 @@ public ResponseEntity create(@Valid @RequestBody Debtors debtors,
             debugMap.put(MESSAGE, "Failed to create debtors resource");
             LOGGER.errorRequest(request, ex, debugMap);
             responseEntity = apiResponseMapper.map(ex);
+        } catch (RestException re) {
+
+            final Map<String, Object> debugMap = new HashMap<>();
```

Comment:
Could you please refactor these lines so that they can be reused .

 final Map<String, Object> debugMap = new HashMap<>();
            debugMap.put(TRANSACTION_ID, transaction.getId());
            debugMap.put(COMPANY_ACCOUNT_ID, companyAccountId);